site/assets/scss/_syntax.scss

```diff
@@ -27,26 +27,25 @@
 // TODO: these values will need to be modified in order to work with Boosted dark mode
 // Right now, values are from Bootstrap from `--base00` to `--baseOF` and the 4 remaining one are copied from Boosted's light mode
 @include color-mode(dark, true) {
-  // --base00: #282c34;
-  // --base01: #353b45;
-  --base02: #3e4451;
-  --base03: #868e96;
-  --base04: #868e96;
-  --base05: #abb2bf;
-  --base06: #b6bdca;
-  --base07: #{$orange-300}; // #d19a66
-  --base08: #{$cyan-300};
-  --base09: #{$orange-300}; // #d19a66
-  --base0A: #{$yellow-200}; // #e5c07b
-  --base0B: #{$teal-300}; // #98c379
-  --base0C: #{$teal-300}; // #56b6c2
-  --base0D: #{$blue-300}; // #61afef
-  --base0E: #{$indigo-200}; // #c678dd
-  --base0F: #{$red-300}; // #be5046
-  --base10: #{$gray-400};
-  --base11: #{$yellow-800};
-  --base12: #{$cyan-900};
-  --base13: #{$pink-800};
+  --base00: #{$gray-950};
+  --base01: #{$gray-500};
+  --base02: #{$teal-300};
+  --base03: #{$cyan-300};
+  --base04: #{$yellow-300};
+  --base05: #{$cyan-400};
+  --base06: #{$gray-400};
+  --base07: #{$pink-800};
+  --base09: #{$green-800};
```

Comment:
We're missing base08, is that fine as is ?

Reply:
Not normal at all, let me check 👀